static/app/views/replays/detail/domMutations/domMutationRow.tsx

```diff
@@ -48,6 +58,12 @@ function DomMutationRow({
     [handleMouseLeave, breadcrumb]
   );
 
+  const handleDimensionChange = useCallback(
+    (path, expandedState, e) =>
+      onDimensionChange && onDimensionChange(index, path, expandedState, e),
+    [index, onDimensionChange]
+  );
```

Comment:
nit: should this be called `handleExpand` or something.

there's two method signatures for `*DimensionChange` so im just thinking aloud about making it clearer which is which